app/assets/stylesheets/layout/users/profile.scss

```diff
@@ -0,0 +1,115 @@
+@import 'master';
+
+
+
+// Mobile
+/*body {
+  @include display(flex);
+  @include flex-direction(column);
+  min-height: 100vh;
+
+  -webkit-font-smoothing: antialiased;
+}*/
+
+
+/*.main-profile {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+*/
+.main-profile {
+    /*position: absolute;*/
+    top: 100px;
+    bottom: 0px;
+    width: 100%;
+    background-image: -webkit-gradient(linear, left top, left bottom, from(#ffffff), to(#ffffff));
+    background-image: -webkit-linear-gradient(top, #ffffff, #ffffff);
+    background-image:    -moz-linear-gradient(top, #ffffff, #ffffff);
+    background-image:      -o-linear-gradient(top, #ffffff, #ffffff);
+    background-image:         linear-gradient(to bottom, #ffffff, #ffffff);
+    display: -webkit-flex;
```

Comment:
the mixins found in the _mixin.scss after. the format goes @include display(flex); // this will do all the vendor prefixes for you